pkg/pb/synthetic_monitoring/checks_extra.go

```diff
@@ -553,6 +610,253 @@ func (s *K6Settings) Validate() error {
 	return nil
 }
 
+func (s *MultiHttpSettings) Validate() error {
+	if len(s.Entries) == 0 {
+		return ErrInvalidMultiHttpTargets
+	}
+
+	if err := validateCollection(s.Entries); err != nil {
```

Comment:
Do we want a max quantity of entries? There is a max of 10 on the frontend I believe, but we can pick anything that makes sense.

Reply:
That's a good point. I'll add that. We will need to work out how to implement per-tenant limits.

Reply:
Done